modules/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer.js

```diff
@@ -98,12 +93,27 @@ export default class GPUScreenGridLayer extends Layer {
     if (changeFlags) {
       this._updateAggregation(changeFlags);
     }
+
+    this._updateColorUniforms(opts);
   }
 
   draw({uniforms}) {
-    const {minColor, maxColor, parameters = {}} = this.props;
-    const {model, maxCountBuffer, cellScale} = this.state;
-    uniforms = Object.assign({}, uniforms, {minColor, maxColor, cellScale});
+    const {parameters = {}} = this.props;
+    const minColor = this.props.minColor || DEFAULT_MINCOLOR;
```

Comment:
Could we use `defaultProps` instead of adding additional logic inside the layer?

Reply:
We need some more work around this before we can do this, we initially planned to deprecated `minColror` and `maxColor`, but during API Audit we decided to keep it until liner interpolation of alpha channel is added. #1946